### DISCUSSION TITLE
Is it possible to abort Quarto rendering based on exit code of pre-render script (or similar approach)?

### DISCUSSION THREAD

**Discussion by yoctobug** [quarto-dev/quarto-cli#6368] [Q&A] [answered]
### Is it possible to abort Quarto rendering based on exit code of pre-render script (or similar approach)?

For example, say we have 

```
project:
  ...
  pre-render:
    - pre-render.py
```

in `_quarto.yml` and the pre-render script terminates with non-zero exit code, _e.g._, with, say, `exit(42)`. Is it possible to automatically abort rendering at that point?

**Comment by mcanouil** [comment, marked as answer]
In a relatively recent update, Quarto is supposed to abort when the pre-render script has non zero exit.

Would you mind testing your script with 1.4 pre-release or at least share a "working" reproducible example?

**Comment by yoctobug** [comment]
OK. I'm at  1.3.433 right now. I'll see about upgrading.

**Reply by yoctobug** [reply]
Confirmed. Upgrading to pre-release 1.4.268 does the trick. Works as expected. TY.
